### ISSUE TITLE
can not found module

### ISSUE THREAD

**Issue by renehankel** [ronald-d-rogers/firebird.embedded#2]
i found your little project and i hoped it can solve my firebird embedded problem.
the thing is if i install my clickonce application(wpf + firebird 3 database) on a new installed windows 7 pc, i get always this error message(sorry is in german but i think you can read):
`Die DLL "fbclient.dll": Das angegebene Modul wurde nicht gefunden. (Ausnahme von HRESULT: 0x8007007E) kann nicht geladen werden.`
i use you PATH tip on program startup, i took over all x86 ddl's also the fbembed.dll, i compile with 'any cpu' option but nothing.
the interesting point is also, on my development pc and on other pc's with my already running application and longer working os system is the embedded version running. but not an on a new system.
any tip from your side?

**Comment by StingyJack** [comment]
I also didn't have this problem until recently when I got a new windows 10 laptop.

**Comment by renehankel** [comment]
my problem was, that i still used the msvcp80.dll and msvcr80.dll files, but it must be the msvcp100.dll and msvcr100.dll files now .

**Comment by StingyJack** [comment]
How did you come to know this?  

I think i figured out how to make it work, but that seems to require I change this solution and any that depend on it to either x86 or x64, and then re-install the package, as well as adding the path code you mentioned.

**Comment by renehankel** [comment]
i found out till know, that you need the following files for an embedded firebird:
fbclient.dll
ib_util.dll
plugins\engine12.dll
all icu*.* files
msvcp*.dll
msvcr*.dll

(please make attention to use the correct bitness – 64 bit files for a 64 bit application and 32 bit files for a 32 bit application)

**Comment by ronald-d-rogers** [comment]
Are you guys saying that the msvp*.dll files are also required now to embed Firebird? And these files are specific to the version of Windows it is being embedded on?

**Comment by renehankel** [comment]
yes! if you have an older computer with some other programs then you have maybe already installed the Microsft Visual C files but not on a new system and when firebird itself has these files in his folder .. i think they can be necessary!

but anyway, on my complete new installed system runs my program with firebird embedded with these files  without error )))

**Comment by renehankel** [comment]
sorry i forgot, if you use 32bit files, i think is not a problem on 64 bit windows, but 64bit files on a 32bis system .... it's maybe a problem. ;) 
i use always 32bit because for me it's ok

**Comment by StewartScottRogers** [comment]
I just started testing embedded firebird in QA and I am getting the same problem. When I install the Firebird Database (Client Only) the problem goes away on my QA server and the database is up and running. My conclusions:  Whatever is needed to fix this problem is in the Client Only install of the Firebird Database 3.0  is missing from the current tooling for this Nuget package.
